internal/controller/dscinitialization/dscinitialization_controller.go

```diff
@@ -421,6 +421,10 @@ func (r *DSCInitializationReconciler) newMonitoringCR(ctx context.Context, dsci
 		defaultMonitoring.Spec.Metrics = dsci.Spec.Monitoring.Metrics
 	}
 
+	if dsci.Spec.Monitoring.Traces != nil {
+		defaultMonitoring.Spec.Traces = dsci.Spec.Monitoring.Traces
```

Comment:
nit: we can avoid the if and just assign the values, eventually we would assign a nil value, but that would be fine